src/targets/gpu/include/migraphx/gpu/miopen.hpp

```diff
@@ -91,12 +97,11 @@ inline convolution_descriptor make_conv(const T& op)
     miopenConvolutionMode_t c_mode = miopenConvolution;
     if(op.group > 1)
         c_mode = miopenGroupConv;
-    std::vector<int> padding(op.padding.begin(), op.padding.end());
-    std::vector<int> stride(op.stride.begin(), op.stride.end());
-    std::vector<int> dilation(op.dilation.begin(), op.dilation.end());
 
-    if(op.kdims() == 1)
-        insert_attribute_1d_dims(padding, stride, dilation);
+    size_t kdims = op.kdims();
+    std::vector<int> padding = get_attributes(op.padding, kdims, 0);
```

Comment:
I meant do something like this:

```cpp
std::vector<int> padding(std::min(2, kdims), 0);
std::copy_backwards(op.padding.begin(), op.padding.end(), padding.end());
```

Reply:
this doesn't work if kdims = 3

Reply:
I was able to fix the above by using `std::max` instead

Reply:
Yea thats right, it should be `std::max`.